editor/editor_settings.cpp

```diff
@@ -1316,8 +1316,44 @@ Ref<ShortCut> ED_GET_SHORTCUT(const String &p_path) {
 	return sc;
 }
 
+struct ShortCutMapping {
+	const char *path;
+	uint32_t keycode;
+};
+
 Ref<ShortCut> ED_SHORTCUT(const String &p_path, const String &p_name, uint32_t p_keycode) {
 
+#ifdef OSX_ENABLED
+	static const ShortCutMapping macos_mappings[] = {
+		{ "editor/play", KEY_MASK_CMD | KEY_B },
```

Comment:
Both have the same key binding(command+b). the line is cloned when the project is run.
https://github.com/godotengine/godot/blob/109242c011255757c981f634a33f59268a113f67/editor/plugins/script_text_editor.cpp#L1648